### PR TITLE
Fixed sliding not playing

### PR DESCRIPTION
**Pull request by PBWiseman**
## Describe your changes

Sliding animation works and ice particles moved to -.9 instead of -1

### CODE REVIEW

**Review by ChaseBENNC9 — APPROVED** [review]
looks good

**Comment by PBWiseman** [comment]
Changed the heat distortion to SortingOrder -5 so it won't clip over the sliding particles

File: Assets/Scripts/Menus/SettingsManager.cs [hunk 1]
```diff
@@ -292,11 +292,11 @@ private void LoadSettings()
         
         try
         {
-            GameSettings.MusicVolume = PlayerPrefs.GetFloat("MusicVolume", .75f); //Default is 75% volume
+            GameSettings.MusicVolume = PlayerPrefs.GetFloat("MusicVolume", .5f); //Default is 50% volume
         }
         catch
         {
-            GameSettings.MusicVolume = .75f; //Default is 75% volume in case the PlayerPrefs value is invalid
+            GameSettings.MusicVolume = .5; //Default is 50% volume in case the PlayerPrefs value is invalid
```
Comment (erokasyuart) [inline Assets/Scripts/Menus/SettingsManager.cs:299]: Add suffix 'f' because unity threw an error about how doubles can't be floats and such.

**Review by erokasyuart — CHANGES_REQUESTED** [review]
Happy with it except for one line of code.

File: Assets/Scripts/Menus/SettingsManager.cs (hunk 1 above)
Comment (PBWiseman) [reply Assets/Scripts/Menus/SettingsManager.cs]: Fixed

**Review by erokasyuart — APPROVED** [review]
Swagalicious my home slice!

**Pull request merged**